controller/array_action/array_mediator_interface.py

```diff
@@ -65,7 +65,7 @@ def copy_to_existing_volume_from_source(self, name, source_name, source_capacity
             source_capacity_in_bytes        : capacity of source to create from
             minimum_volume_size_in_bytes    : if source capacity is lower than this value volume will
                                             be increased to this value
-            pool_id                         : pool of the volume and source object to find them more efficiently.
+            pool                            : pool of the volume and source object to find them more efficiently.
```

Comment:
is it still always the same pool for both sides?

Reply:
in ds8k - yes
elsewhere - no 
there wasn't any work done there, the work now was focused on `snapshot`
I think it possible to do the same thing with `clone` but I don't know if we want to support it because by spec it needs to use the [same storageclass](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/#introduction)...

Reply:
I'm asking only in snapshot context (creating a volume from snapshot)

Reply:
so no, not always

Reply:
but its not what you refered

Reply:
I opened ticket CSI-2937